index.js

```diff
@@ -190,7 +180,7 @@ module.exports = Hermes;
 Hermes.prototype.publish = function (queueName, data) {
   /*jshint maxcomplexity:7 */
   debug('hermes publish', queueName, data);
-  if (!~this.queues.indexOf(queueName)) {
+  if (!~this.opts.queues.indexOf(queueName)) {
     throw new Error('attempting to publish to invalid queue: '+queueName);
   }
```

Comment:
not sure how helpful it would be. I could reduce 3 lines into 1 with it.
`_assertValidQueueName(queueName, 'attempting to publish to invalid queue: %s')`